src/components/containers/EventCard/EventCard.tsx

```diff
@@ -22,25 +24,22 @@ export const EventCard: React.FC<EventCardProps> = ({
     startDate,
     locationName,
 }) => {
+    // color fallback
+    if (!color) color = "#4340BE"
```

Comment:
Can you extract this color to consts or somewhere else?

Reply:
i dunno, we don't have any kind of project pallet yet
so making one, when some thing are going to change seems not worth effort (in this case)

Reply:
ok then